test/ng/qSpec.js

```diff
@@ -2113,7 +2113,7 @@ describe('q', function() {
       });
 
 
-      it('should log exceptions throw in a progressack and stop propagation, but shoud NOT reject ' +
+      it('should log exceptions thrown in a progressback and stop propagation, but should NOT reject ' +
```

Comment:
This was the only instance of `progressack`, I assumed it was a typo for `progressback`. I could be wrong.